.circleci/config.yml

```diff
@@ -164,7 +164,7 @@ executors:
       BASH_ENV: /etc/profile ## Ref: https://circleci.com/docs/env-vars/#alpine-linux
       NVM_ARCH_UNOFFICIAL_OVERRIDE: x64-musl ## Ref: https://github.com/nvm-sh/nvm/issues/1102#issuecomment-550572252
     docker:
-      - image: node:lts-alpine # Ref: https://hub.docker.com/_/node?tab=tags&page=1&name=alpine
+      - image: node:18-alpine3.19 # Ref: https://hub.docker.com/_/node?tab=tags&page=1&name=alpine
```

Comment:
If it works using node:18.17.1-alpine, then I suggest using that image just to align with nvmrc file.

Reply:
Unfortunately node 18.17.1 is only released with alpine 3.18, which has more vulnerabilities than 3.19.
I think it is better to upgrade to latest node 18 and use alpine 3.19, known to work.
See https://hub.docker.com/_/node/tags?name=18.17.1-alpine vs https://hub.docker.com/_/node/tags?name=18.20.3-alpine3.19